docs/overview.md

```diff
@@ -1,26 +1,116 @@
+## How does parsing work?
+
+The MyST Document engine knows how to parse many kinds of documents into {term}`MyST AST`. This is most-commonly done with markdown files (`.md`) or Jupyter Notebooks (`.ipynb`) written in {term}`MyST Markdown`, a flavor of markdown that was designed for the MyST Document Engine.
+
+However, the MyST Engine knows how to parse other kinds of syntax into MyST AST as well. For example, [for admonition compatibility with GitHub Markdown](admonition-github-compatibility). This is because we see the {term}`MyST AST` as the primary point of _standardization_ for the MyST ecosystem, not the Markdown flavor. Input documents may have many different forms, but once it is parsed into {term}`MyST AST`, the document should have a standardized structure defined by the {term}`MyST Specification`.
+
+## What is the MyST AST and Specification?
+
+The {term}`MyST AST` is a version of parsed MyST content that follows a specific structure so that Renderers can convert into different outputs. MyST AST conforms to the {term}`MyST Specification`. The {term}`MyST Specification` is a superset of [the `mdast` specification](https://github.com/syntax-tree/mdast) (meaning that MyST includes all of the syntax that `mdast` supports, plus some extra ones like Directives).
+
+Here's an example of MyST content, and the same content parsed as MyST AST:
+
+````{list-table}
+:header-rows: 1
+- - Content
+  - AST
+- - `**Hello there!**`
+  - ```yaml
+    - type: block
+    children:
+      - type: paragraph
+        children:
+          - type: strong
+            children:
+              - type: text
+                value: Hello there!
+    ```
+````
+
+The easiest way to understand how {term}`MyST Markdown` gets converted to {term}`MyST AST` is to [explore the MyST Sandbox](https://mystmd.org/sandbox). Click the {kbd}`AST` tab to see the underlying AST structure for anything that you type. The {kbd}`PRE` tab represents the initial parsing phase, and the {kbd}`POST` tab represents the AST after the resolving phase.
+
+(overview-transformers)=
+## How do MyST Transformers work?
+
+MyST Transformers are a way to convert a {term}`MyST AST` node into another type of node. Transformers operate on AST rather than on raw Markdown because AST has more standardized structure to work with.
+
+For example, consider a Markdown link like `[some text](#a-label)`. In MyST Markdown, this defines a **cross-reference** to `#a-label`, but it uses Markdown link syntax. We use a MyST Transformer to convert that Markdown to a cross-reference AST node like so:
+
+- First parse the Markdown `[some text](#a-label)`.
+- The result is a MyST AST node for a **Markdown link**.
+- Next, search the document AST for any Markdown link nodes with a target that starts with `#`. Check if there's [a target](./cross-references.md) for the link. If so, it is meant to be a cross reference.
+- If so, run a **Transformer** that converts the Markdown Link node into a Cross Reference node.
```

Comment:
Just to make sure I understand: the rendered runs the transformer? Or there's a Big Transformer that runs the Little Transformers?